src/clusterfuzz/_internal/bot/tasks/utasks/fuzz_task.py

```diff
@@ -1962,6 +1963,13 @@ def postprocess(self, uworker_output):
       data_types.FuzzTargetsCount(
           id=uworker_input.job_type, count=new_targets_count).put()
 
```

Comment:
See ~10 lines above there's a bailout for environment.is_engine_fuzzer_job() which explains why we not see any engine fuzzer stats any more after this commit.